pkg/heat/funcs.go

```diff
@@ -34,3 +36,18 @@ func GetHeatSecurityContext() *corev1.SecurityContext {
 		},
 	}
 }
+
+func FormatProbes(port int32) *corev1.Probe {
```

Comment:
Normally liveness probes have higher `failureThreshold`, though we use the same(default 3) here. If we're creating a common function to reduce code verbosity, we could add a probleHandler argument and use it for heat-engine as well?